test/files/emails/quota-exceeded.txt

```diff
@@ -1,11 +1,8 @@
-HELO mail.external.tld
-MAIL FROM: user@external.tld
-RCPT TO: quotauser@otherdomain.tld
-DATA
-From: Docker Mail Server <user@external.tld>
+From: Docker Mail Server <example-user@example.test>
 To: Existing Local User <quotauser@otherdomain.tld>
 Date: Sat, 22 May 2010 07:43:25 -0400
-Subject: Test Message saturating your mailbox
+Subject: Test message saturating your mailbox
+
```

Comment:
Unnecessary line break introduced between subject and body? (_change is not consistent with other files_)